pytrustnfe/nfe/danfce.py

```diff
@@ -125,21 +135,19 @@ def ide_emit(self, oXML=None):
     def danfce_information(self, oXML=None):
         el_ide = oXML.find(".//{http://www.portalfiscal.inf.br/nfe}ide")
         tipo_emissao = tagtext(oNode=el_ide, cTag='tpEmis')
+        self.drawTitle(
+            "DANFE NFC-e - Documento Auxiliar da Nota Fiscal de",
+            7, 'NimbusSanL-Bold')
+
+        self.drawTitle("Consumidor Eletrônica", 7, 'NimbusSanL-Bold')
+
+        self.drawString(
+            "NFC-e não permite aproveitamento de crédito de ICMS", True)
```

Comment:
Essa mensagem tu não tinha dito que estava errada aqui?

Reply:
self.drawString("NFC-e não permite aproveitamento de crédito de ICMS", True)
isso aki deixou de ser obrigatório para alguns estados e fica como parâmetro opcional, meio que não lembra disso.
como demostra nessa questão aki http://boletim.fazenda.pr.gov.br/boletins/item/2017/2

Reply:
se for o "DANFE NFC-e - Documento Auxiliar da Nota Fiscal de Consumidor Eletrônica" ela fica incorreta se ela não sair junto com a mensagem de contingencia que acontecia por causa do if que removi